tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -28,9 +28,20 @@ public CIVisibilitySettings(IConfigurationSource source, IConfigurationTelemetry
             var config = new ConfigurationBuilder(source, telemetry);
             Enabled = config.WithKeys(ConfigurationKeys.CIVisibility.Enabled).AsBool();
             Agentless = config.WithKeys(ConfigurationKeys.CIVisibility.AgentlessEnabled).AsBool(false);
+            Site = config.WithKeys(ConfigurationKeys.Site).AsString("datadoghq.com");
+
+            if (Enabled == false)
```

Comment:
Out of interest, why do we need to load site before this? If some of this config _does_ need to be loaded explicitly before Enabled, it might be worth mentioning that, just so that noone moves it later and breaks something

Reply:
meh, I did that just because `Site` is not nullable and I don't want it to have a string.Empty value.